### DISCUSSION TITLE
WMF to PDF [ImageMagick.MagickMissingDelegateErrorException: no decode delegate for this image format `WMF']

### DISCUSSION THREAD

**Discussion by viktorshevchenko210** [dlemstra/Magick.NET#1183] [Help]
There is an error converting WMF to PDF:
```
ImageMagick.MagickMissingDelegateErrorException: no decode delegate for this image format `WMF' @ error/constitute.c/ReadImage/572
```

Property MagickNET.Delegates returns following delegates
```
bzlib cairo fontconfig freetype heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png raqm raw rsvg tiff webp xml zip zlib
```

Installing ImageMagick from official site it is possible to add wmf delegate. File is converted successfully using installed library. But C# application doesn't see that delegate.

How to add wmf delegate to library?

**Comment by dlemstra** [comment]
WMF and EMF are supported under Windows but on other platforms this requires the use of an external library. This library is licensed under the GPL license and not included by default. In theory you could probably add this yourself to the Magick.Native library but because of the used license I cannot add this for you.
